apps/api/src/app/order/order.service.ts

```diff
@@ -194,7 +194,63 @@ export class OrderService {
     return order;
   }
 
-  public async deleteOrders(where: Prisma.OrderWhereInput): Promise<number> {
+  public async deleteOrders(where: Prisma.OrderWhereInput, filters: Filter[]): Promise<number> {
+
+    const {
```

Comment:
Instead of duplicating this filter logic, I'd suggest to call `getOrders()`, map the `id`’s and call `deleteOrders()` with an `in` query including the `id`’s.